src/pages/cheoljinPark/Login/LoginForm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import './LoginForm.scss';
+
+function LoginForm({
+  loginId,
+  loginPw,
+  loginCheck,
+  setLoginId,
+  setLoginPw,
+  setLoginCheck,
+}) {
+  const navigate = useNavigate();
+
+  const handleIdInput = event => {
+    setLoginId(event.target.value);
+  };
+  const handlePwInput = event => {
+    setLoginPw(event.target.value);
+  };
+
+  const loginInspector = () =>
+    loginId.includes('@') && loginPw.length > 4
+      ? setLoginCheck(true)
+      : setLoginCheck(false);
+
+  const sendMain = () =>
+    loginCheck ? navigate('/main-park') : alert('정신 차리세요');
+
+  return (
+    <form className="login-form">
+      <input
+        type="text"
+        className="login-id loginInput"
+        placeholder="전화번호, 사용자 이름 또는 이메일"
+        onChange={handleIdInput}
+        onKeyUp={loginInspector}
+      />
+      <input
+        type="password"
+        className="login-pw loginInput"
+        placeholder="비밀번호"
+        onChange={handlePwInput}
+        onKeyUp={loginInspector}
+      />
+      <button
+        className="login-btn"
+        onClick={sendMain}
+        style={
+          loginCheck
+            ? { opacity: 1, cursor: 'pointer' }
+            : { opacity: 0.5, cursor: 'not-allowed' }
+        }
```

Comment:
className의 동적 사용을 적용하면 좋을 것 같아요
인라인 스타일의 우선 순위가 높아서, 팀원이 class를 수정했을 때
class가 적용이 안되는 문제가 있다고 해요.

조건 연산자를 활용해서 className을 동적으로 쓸 수 있음
className={원래 클래스 +활성화 클래스 <-- 조건 연산자로 클래스 추가 여부 결정}
```suggestion
       className={`loginBtn ${
              idValue.indexOf('@') > -1 && pwValue.length >= 5
                ? 'loginBtnLive'
                : ''
            }`}
```

Reply:
good 클래스로 처리했읍니다..